### ISSUE TITLE
"잠재능력 재설정 불가" variable missing newline

### ISSUE THREAD

**Issue by Sunaries** [KENNYSOFT/WzComparerR2#5]
Greetings,

![ingame](https://cdn.discordapp.com/attachments/236180502200057856/266252130246066188/dominatorpendant.png)
↑ The above image is an in-game version. As you can see there is a new line after "Unique Equipped Item"

![current version](https://cdn.discordapp.com/attachments/236180502200057856/266252114831867914/1122372.png)
↑ Current version of yours does not seem to do this.

P.S. I also noticed that the potential says Legendary (L) while in-game says Unique (U).  Error?

P.P.S: 

![test](https://cdn.discordapp.com/attachments/236180502200057856/266262675841875968/AbsoLab_fixed_PotentialKR.png)

Though it does not have a newline for this item. Hmm...

**Comment by KENNYSOFT** [comment]
Thanks for reporting. However, I already noticed these bugs, also in same item. My comment (in Korean) for this item can be found in 인소야닷컴(http://www.insoya.com/bbs/zboard.php?id=maple_info&no=35668).

The first bug is because of the orange properties length. The properties doesn't make newline by its property itself(like 잠재능력 재설정 불가), but make newline if whole properties line is longer than some specific value(this version has `255` in `GearTooltipRender2.cs:216`). So the first one is easily resolved by adjusting that value.

The second bug has relation with "fixedGrade" value, which is 5 for this item. However, I have searched for fixedGrade value in whole item of KMS, the fixedGrade value is only in 2, 3, 5, which is rare(blue) color, epic(purple) color, and unique(yellow) color, respectively. Besides, this program assumes the fixedGrade value 2, 3, 4, 5 as rare, epic, unique, and legendary(in `Gear.cs:572-575`), respectively. If I have some different values for fixedGrade(especially about in-game legendary fixed item), than I should fix this bug.

Also, I will fix the ExclusiveEquip list and SetItemTooltip to not show same named items duplicated.
